week-5/A-dom-manipulation/exercise.js

```diff
@@ -22,12 +54,57 @@ Task 3
 
 When a user clicks the 'Add some text' button, a new paragraph should be added below the buttons that says "Read more below."
 */
+// var addTextBtn = document.querySelector("#addTextBtn");
+// addTextBtn.addEventListener("click", addText);
 
+////////////METHOD 2//////////
+divButtonChild[2].addEventListener("click", addText);
+
+function addText() {
+    var paragraph = document.createElement('p'); // here we're just creating it, element is not visible yet
+    addTextBtn.appendChild(paragraph);
+    paragraph.innerText = "Read me below.";
+    addTextBtn.removeEventListener("click", addText); // to stop at one paragraph
+    addTextBtn.style.backgroundColor = "Yellow"; //Just an extra
+}
 
 
 /*
 Task 4
 ======
 
 When the 'Larger links!' button is clicked, the text of all links on the page should increase.
-*/
\ No newline at end of file
+*/
+//////////METHOD 1///////////
+// var largerLinksBtn = document.querySelector("#largerLinksBtn");
+// largerLinksBtn.addEventListener("click", largeLinks);
+
+// function largeLinks() {
+//     var large = document.querySelectorAll(".btn-lrg");
+//     large.forEach(function (element) {
+//         if (element.style.fontSize == "xx-large") {
+//             element.style.fontSize = "medium";
+
+//         } else {
+//             element.style.fontSize = "xx-large";
+//         }
+//     });
+
+// }
```

Comment:
Nice idea to make it toggle between xx-large and medium.  (Perhaps you could update the button's text so it is clear what will happen when you next press it.)